### PR TITLE
Feature/s3 creds

### PR DESCRIPTION
**Pull request by robert-bryson**
Related to https://github.com/GSA/data.gov/issues/3648

These darn things keep getting blown away. I just want to rebase, not sure why there are so many old commits?

### CODE REVIEW

File: proxy/nginx.conf [hunk 1]
```diff
@@ -29,6 +29,10 @@ http {
     listen {{port}};
     root public;
     index index.html index.htm Default.htm;
+
+    location = /robots.txt {
+        alias /public/robots.txt;
```
Comment (robert-bryson) [inline proxy/nginx.conf:34]: @btylerburton, I feel like you've been in the `nginx.conf` files more recently than I have, does this seem like the right place to drop this?

Comment (robert-bryson) [reply proxy/nginx.conf:34]: It is within (what I think is?) the first hit server block.

`nginx-common.conf` appears to be hit after `auth` stuff is, I think. I don't think we want `robots.txt` or the sitemaps to be behind the auth, right? I might not be understanding how auth is working in this case, though.

Comment (btylerburton) [reply proxy/nginx.conf:34]: check out this block then: https://github.com/GSA/catalog.data.gov/blob/main/proxy/nginx-common.conf#L19

we wanted health check not to be backed by auth and this is working

**Comment by robert-bryson** [comment]
I might need some additional testing on this (the routing and the schedule action) once merged, but some of the stuff needs to be in to do it.

File: proxy/nginx.conf (hunk 1 above)
Comment (robert-bryson) [reply proxy/nginx.conf]: I dropped it down by the `500` block. Thanks for this!

**Review by nickumia-reisys — APPROVED** [review]
⛓️

**Pull request merged**